src/components/GeographyMap/index.tsx

```diff
@@ -2,16 +2,80 @@ import { MapContainer, GeoJSON, TileLayer } from 'react-leaflet';
 import { LatLng, geoJSON, latLng } from "leaflet";
 import 'leaflet/dist/leaflet.css';
 import { Paper } from '@mantine/core';
+import { parseValue } from "~/util/surrealql";
+import { GeometryCollection, GeometryLine, GeometryMultiLine, GeometryMultiPoint, GeometryMultiPolygon, GeometryPoint, GeometryPolygon } from "surrealdb.js";
+
+export type GeographyInput =
+	| GeometryPoint
+	| GeometryLine
+	| GeometryPolygon
+	| GeometryMultiPoint
+	| GeometryMultiLine
+	| GeometryMultiPolygon
+	| GeometryCollection<any>;
+
+const toGeoPoint = (data: GeometryPoint) => {
+	const [long, lat] = data.point;
+	return [long.toNumber(), lat.toNumber()] as const;
+};
+
+const toGeoJSONObject = (input: GeographyInput): any => {
```

Comment:
@Odonno you can simply call `.toJSON()` on the Geometry instance and it will generate the correct JSON-like object for you 😃